web/cobrands/isleofwight/assets.js

```diff
@@ -99,20 +99,24 @@ fixmystreet.assets.add($.extend(true, {}, defaults, {
     })
 }));
 
+function found_item(layer, asset) {
+  var id = asset.attributes.asset_id || '';
```

Comment:
This was different in the two calls, central_asset_id/asset_id.

Reply:
it should be `central_asset_id`, not everything has an `asset_id`